src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,77 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 20px;
+  width: 340px;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
+}
+
+.buttonWrapper {
+  margin-left: 38px;
+}
+
+.buttonWrapper > a:hover {
+  top: 0;
+}
+
+.inputText {
+  width: 291px;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+    grid-template-columns: 1fr;
```

Comment:
There's no need to set `grid-template-columns` if you don't have a grid element, so you can just remove this property.

Reply:
ok